celi_framework/utils/anthropic_client.py

```diff
@@ -15,15 +15,26 @@
 
 @functools.lru_cache()
 def get_anthropic_bedrock_client():
+
+    args={}
+    #Default the AWS region to us-west-2
+    try:
+        if aws_region is None:
+            aws_region = os.environ.get("AWS_REGION") or "us-west-2"
+            logging.info("AWS region not defined. Defaulting to 'us-west-2'.")
+            args = {"aws_region": aws_region}
+    except NameError:
+        aws_region = os.environ.get("AWS_REGION") or "us-west-2"
+        logging.info("AWS region not defined. Defaulting to 'us-west-2'.")
+        args = {"aws_region": aws_region}
```

Comment:
I think here, aws_region is always undefined, so I think this block should be:

aws_region = os.environ.get("AWS_REGION", None)
if aws_region is None:
        aws_region = 'us-west-2'
        logging.info(f"AWS region not defined. Defaulting to {aws_region}.")
args = {"aws_region": aws_region}